irohad/main/application.cpp

```diff
@@ -129,6 +133,7 @@ void Irohad::init() {
   // Recover WSV from the existing ledger to be sure it is consistent
   initWsvRestorer();
   restoreWsv();
+  updatePeers();
```

Comment:
so this could be 
```suggestion
  if (opt_alternative_peers) {
    resetPeers(*opt_alternative_peers);
  }
```

Reply:
I miss you idea here, `resetPeers` just removes all peers from WSV, why should we pass something to it?

Reply:
i mean a function `rasetPeers` that replaces the current peers with the ones provided in tis argument. It would do so unconditionally, i.e. regardless the collection empty state, to make thing simplier. And the need to replace peers is provided by the optional `opt_alternative_peers`, which is set when the user passed the corresponding command line flag.

Reply:
I guess emptiness check is still necessary since without it everything becomes broken and broken in a pretty unobvious manner (like failed assertions in random places).

Reply:
this check is done by you in `idohad.cpp`, isn't it?

Reply:
Although I would encourage another assert here.

Reply:
Well, I did the same mistake yesterday :). Irohad.cpp can check everything but we have ITF as well.